src/modules/navigator/navigator_params.c

```diff
@@ -128,28 +128,24 @@ PARAM_DEFINE_FLOAT(NAV_MC_ALT_RAD, 0.8f);
 PARAM_DEFINE_INT32(NAV_TRAFF_AVOID, 1);
 
 /**
- * Set NAV TRAFFIC AVOID RADIUS MANNED
+ * Set avoidance radius/distance for manned vehicles (ADS-B/FLARM).
```

Comment:
I would keep this relatively short for QGC.

![Screenshot from 2020-02-02 12-52-07](https://user-images.githubusercontent.com/84712/73612583-e1a2d500-45ba-11ea-9c5c-ee4bd5884e5e.png)

Reply:
Could perhaps change these to

> Set ADS-B/FLARM avoidance radius.

 and 
> 
> Set UTM avoidance radius.

The extended description covers the detail.

Reply:
I have no opinion on the optimal solution